config/routes.rb

```diff
@@ -408,7 +408,7 @@ def external_redirect(path_prefix, target)
     get "/placeholder" => "placeholder#show", as: :placeholder
   end
 
-  get "/courts-tribunals(.:locale)", as: "courts", to: "organisations#index", courts_only: true, constraints: { locale: valid_locales_regex }
+  # TODO: the organisations controller has been removed but this route is still required to get the relevant helper methods. This can be removed once new helpers have been created.
```

Comment:
Not sure I fully understand why we still need this, but should we fix this TODO as part of this PR?

Reply:
This is a longstanding issue when rendering has been migrated out of Whitehall, as other parts of Whitehall use the path helpers that Rails generates when routes exist (e.g. `organisations_path` in this case).

There are multiple other examples, e.g. [for get-involved](https://github.com/alphagov/whitehall/blob/42d3182d97ff2d28e6140c4f6eabc09f527620e6/config/routes.rb#L68-L72), so we have a [separate Trello card](https://trello.com/c/EfhUs8ak/33-tidy-whitehalls-routes-file) to sort out this issue by creating our own helper methods and then removing them from the routes file.